### ISSUE TITLE
Docker image eclipse/zenoh-bridge-dds:master for arm64 is broken.

### ISSUE THREAD

**Issue by nayakned** [eclipse-zenoh/zenoh-plugin-dds#43]
Attempts to execute the arm64 docker for eclipse/zenoh-bridge-dds:master on Raspberry Pi 4 with Ubuntu fails.

Steps to reproduce on Raspi 4
```
docker pull eclipse/zenoh-bridge-dds:master
docker run -it eclipse/zenoh-bridge-dds:master
```

Result: 
```
/usr/local/bin/zenoh-bridge-dds: /lib/aarch64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /usr/local/bin/zenoh-bridge-dds)
```

libc-2.28.so is available in the /lib/aarch64-linux-gnu/, but apparently v2.29 is required.
Interestingly, this was not observed in the amd64 docker.

**Comment by JEnoch** [comment]
Hi @nayakned ,

It looks like the binary have been cross-compiled in an image where `libc` version is 2.29, while the runtime image is a `debian:buster-slim` with `libc` version 2.28.
My guess is that the cause is in the Dockerfile where the builder is a `rust:slim` image that might have recently changed of base image. I just pushed a fix attempt (d542230). After [this action](https://github.com/eclipse-zenoh/zenoh-plugin-dds/actions/runs/1234335652) finished (in about 1.5 hour), please pull `eclipse/zenoh-bridge-dds:master ` again and let me know if it works better.

**Comment by nayakned** [comment]
Thanks for the quick response. 
The issue is resolved.
